lib/gooddata/lcm/lcm2.rb

```diff
@@ -342,19 +345,25 @@ def perform(mode, params = {})
           results << res
         end
 
-        # Fail whole execution if there is any failed action
-        fail(JSON.pretty_generate(errors)) if strict_mode && errors.any?
-
         brick_results = {}
         actions.each_with_index do |action, index|
           brick_results[action.short_name] = results[index]
         end
 
-        {
+        result = {
           actions: actions.map(&:short_name),
           results: brick_results,
-          params: params
+          params: params,
+          success: errors.empty?
         }
+
+        # Fail whole execution if there is any failed action
+        fail(JSON.pretty_generate(errors)) if strict_mode && errors.any?
```

Comment:
There is a risk of regression here. We should make sure that the failing mechanism is covered with unit tests.

Reply:
Done, see #1272 please.